packages/trace-viewer/src/ui/sourceTab.tsx

```diff
@@ -29,8 +29,9 @@ export const SourceTab: React.FunctionComponent<{
   action: ActionTraceEvent | undefined,
   sources: Map<string, SourceModel>,
   hideStackFrames?: boolean,
+  rootDir?: string,
   fallbackLocation?: StackFrame,
-}> = ({ action, sources, hideStackFrames, fallbackLocation }) => {
+}> = ({ action, sources, hideStackFrames, rootDir, fallbackLocation }) => {
```

Comment:
File name in sources is rendered relative to root, not testDir, as in Suite titles in @playwright/test